bindings/python/expose-ldlt.cpp

```diff
@@ -20,10 +23,19 @@ struct LDLTVisitor : bp::def_visitor<LDLTVisitor<LDLTtype>> {
     return fac.solveInPlace(rhsAndX);
   }
 
+  template <typename PlainObject>
+  static PlainObject solve(const LDLTtype &fac, Eigen::Ref<PlainObject> rhs) {
```

Comment:
Why you don't add directly the solve method in `DenseLDLT`, `BunchKaufman` and `ProxSuiteLDLTWrapper` ?

The method will also be available in C++ then.

Reply:
So you can add it to `DenseLDLT`, `BlockLDLT` and `ProxSuiteLDLTWrapper`. Since `BunchKaufman` already inherit from `SolverBase` the method already exists.

Reply:
Alright, I'll do that!